apps/src/templates/curriculumCatalog/CurriculumCatalogCard.jsx

```diff
@@ -87,7 +93,17 @@ const CustomizableCurriculumCatalogCard = ({
     <img src={imageSrc} alt={imageAltText} />
     <div className="curriculumInfoContainer">
       {/*TODO [MEG]: Show all subjects and topics rather than only the first one */}
-      <p className="overline">{subjectsAndTopics[0]}</p>
+      <div className="tagsAndTranslatabilityContainer">
+        <p className="overline">{subjectsAndTopics[0]}</p>
+        {/*TODO [MEG]: Ensure this icon matches spec when we update FontAwesome */}
```

Comment:
I am curious, what's the timeline for the fontawesome update?

Reply:
Good question. Just inquired about it here https://codedotorg.slack.com/archives/C046G4TRLEN/p1680640663555569?thread_ts=1678710322.052629&cid=C046G4TRLEN

It looks like Darin's been working on it recently